src/rpc/ws_client/subscription.rs

```diff
@@ -44,7 +44,7 @@ impl<Notification: DeserializeOwned> HandleSubscription<Notification>
 			// Sender was disconnected, therefore no further messages are to be expected.
 			Err(_) => return None,
 		};
-		// Decode to the notification on is expecting.
+		// Deserialze to the notification one is expecting.
```

Comment:
Still a typo ;) But honestly, I doubt that the comment is adding any value anyhow.

Reply:
Agreed. Removed